src/component/MenuPanel.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import MenuItem from './MenuItem';
+import styles from '../generalStyle.css';
+
+const MenuPanel = () => (
+  <div className={styles.menuPanelStyle}>
+    <MenuItem text="People" />
```

Comment:
Probably in the near feature, we should put this hard-coded strings like 'People', in a constant file.